apps/srt-tunnel.cpp

```diff
@@ -534,7 +534,7 @@ class TcpMedium: public Medium
         tcp_close(m_socket);
         m_socket = -1;
     }
-    virtual void CloseInternal() { return CloseTcp(); }
+    virtual void CloseInternal() override { return CloseTcp(); }
```

Comment:
There's likely another PR that fixes some logical errors and this one among others. Not sure if this should really be fixed in this PR.

Reply:
"likely" is not a good term here, as this warning has to be fixed. I just didn't want to create a separate PR for such a small change. But it will go in a separate commit.